src/components/shared/Entity/ValidationErrorSummary/EndpointConfigErrors.tsx

```diff
@@ -1,16 +1,18 @@
 import SectionError from 'components/shared/Entity/ValidationErrorSummary/SectionError';
 import {
     useEndpointConfigStore_endpointConfigErrors,
+    useEndpointConfigStore_endpointCustomErrors,
     useEndpointConfigStore_endpointSchema,
 } from 'stores/EndpointConfig';
 
 function EndpointConfigErrors() {
+    const endpointCustomErrors = useEndpointConfigStore_endpointCustomErrors();
     const endpointErrors = useEndpointConfigStore_endpointConfigErrors();
     const endpointSchema = useEndpointConfigStore_endpointSchema();
 
     return (
         <SectionError
-            errors={endpointErrors}
+            errors={endpointCustomErrors.concat(endpointErrors)}
```

Comment:
Made custom errors display first as I am hoping Users will read that first and fix OAuth issues.